openlibrary/templates/home/categories.html

```diff
@@ -0,0 +1,107 @@
+$def with (id="ol-carousel")
+
+<div class="head">
+  <h2 class="inline"><a href="/read">Browse By Category</a></h2>
+  <p class="inline sansserif large collapse">
+  </p>
+</div>
+
+<script type="text/javascript">
+  \$(function() {
+
+    var Subject = {
+        count: function (subject, callback) {
+            var url = '/subjects/' + subject + '.json';
+            \$.ajax({
+                type: "GET",
+                url: url,
+		dataType: "json",
```

Comment:
Looks like the editor is mixing tabs and spaces. Other misalignments below. Let's use all spaces.